src/Markdown.MAML/Transformer/ModelTransformerBase.cs

```diff
@@ -165,33 +171,44 @@ protected MamlExample ExampleRule()
             // ```
             // Remarks
             var node = GetNextNode();
-            var headingNode = GetHeadingWithExpectedLevel(node, EXAMPLE_HEADING_LEVEL);
-            if (headingNode == null)
+            try
             {
-                return null;
-            }
+                var headingNode = GetHeadingWithExpectedLevel(node, EXAMPLE_HEADING_LEVEL);
 
-            MamlExample example = new MamlExample()
-            {
-                Title = headingNode.Text
-            };
-            example.Introduction = GetTextFromParagraphNode(ParagraphNodeRule());
-            CodeBlockNode codeBlock;
-            while ((codeBlock = CodeBlockRule()) != null)
-            {
-                if (example.Code == null)
+                if (headingNode == null)
                 {
-                    example.Code = codeBlock.Text;
+                    return null;
                 }
-                else
+
+                MamlExample example = new MamlExample()
+                {
+                    Title = headingNode.Text
+                };
+                example.Introduction = GetTextFromParagraphNode(ParagraphNodeRule());
+                CodeBlockNode codeBlock;
+                while ((codeBlock = CodeBlockRule()) != null)
                 {
-                    example.Code += "\r\n\r\n" + codeBlock.Text;
+                    if (example.Code == null)
+                    {
+                        example.Code = codeBlock.Text;
+                    }
+                    else
+                    {
+                        example.Code += "\r\n\r\n" + codeBlock.Text;
+                    }
                 }
+
+                example.Remarks = GetTextFromParagraphNode(ParagraphNodeRule());
+
+                return example;
             }
+            catch (HelpSchemaException headingException)
+            {
+                Report("Schema exception. This can occur when there are multiple code blocks in one example. " + headingException.Message);
```

Comment:
Do not use + for string concatenation. Use String.Format("Schema exception. This can occur when there are multiple code blocks in one example. {0}", headingException.Message) instead.